accounts/keys.go

```diff
@@ -0,0 +1,168 @@
+package accounts
+
+import (
+	"crypto/ecdsa"
+	"errors"
+	"io"
+	"os"
+
+	"github.com/howeyc/gopass"
+)
+
+var (
+	errNoKeystoreDir = errors.New("Keystore directory does not exists")
+)
+
+// PassPhraser is interface for retrieving
+// pass phrase for Eth keys
+//
+// If you want to retrieve pass phrases reader different ways
+// (e.g: from file, from env variables, interactively from terminal)
+// you must implement PassPhraser reader a different way and pass it to
+// KeyOpener instance
+type PassPhraser interface {
+	GetPassPhrase() (string, error)
+}
+
+// KeyOpener is interface for loading Eth keys
+type KeyOpener interface {
+	// GetPassPhraser return PassPhraser interface
+	// that provides pass phrase for loaded keys
+	GetPassPhraser() PassPhraser
+	// OpenKeystore opens key storage.
+	OpenKeystore() (bool, error)
```

Comment:
Why is there a bool result? "We couldn't open a keystore, but whatever, we don't know how it happened"?

Reply:
A bool flag shows that keystore was created (false if just opened). Need to documenting that.